### PR TITLE
[Tick-09]

### PR DESCRIPTION
**Pull request by Ibrax9760**

### CODE REVIEW

File: contact.html
Comment (woolffangg) [inline contact.html:1]: not good trailing space

File: contact.html [hunk 1]
```diff
@@ -148,11 +148,13 @@ <h3>Adresse</h3>
...
-						</div>
-						<h3>Email</h3>
-						<p><br><br>contact@pyrenees-software.com</p>
+                        <div class="address-icon">
+                            <i class="fa fa-envelope" aria-hidden="true"></i>
+                        </div>
+                        <h3>Email</h3>
+                        <p><a href="mailto:contact@pyrenees-software.com">contact@pyrenees-software.com</a></p>
+                    </div>
+
```
Comment (xavierarce) [inline contact.html:157]: Unnecessary extra line

**Review by woolffangg — CHANGES_REQUESTED** [review]
Very bad code, too much trailing space

File: contact.html [hunk 2]
```diff
@@ -148,11 +148,13 @@ <h3>Adresse</h3>
...
 					<div class="address-item">
-						<div class="address-icon">
-							<i class="fa fa-envelope" aria-hidden="true"></i>
-						</div>
-						<h3>Email</h3>
-						<p><br><br>contact@pyrenees-software.com</p>
+                        <div class="address-icon">
+                            <i class="fa fa-envelope" aria-hidden="true"></i>
+                        </div>
+                        <h3>Email</h3>
```
Comment (xavierarce) [inline contact.html:154]: Bad indentation

**Review by xavierarce — CHANGES_REQUESTED** [review]
Bad indentation

**Review by fabienmarotte — CHANGES_REQUESTED** [review]
Can you fix the indentations issues ?

**Pull request closed without merging**